swarm/regimes.py

```diff
@@ -24,13 +24,14 @@ def make_bee(regime, x, y, *args, **kwargs):
     and share resources
     Also optional to use
     """
-    return functools.partial(regime, x, y, *args, **kwargs)
+    thestrkwargs = {key: str(value) for key, value in kwargs.items()}
```

Comment:
Yeah, I get it.  But I'm caught in this awkward place where the way I've managed recording attributes previously in state machines I've built has been with classes, where such a dictionary has been an attribute of a class. And you're doing all these things with closures which I'm pretty unfamiliar with. I'm just trying to use as much of your stuff in a way that let's me get some results.  I'll settle for leaving my ugly make bee amendment in my experiments folder. And I still appreciate the coding feedback too. I'll get there eventually hope. Keep pointing these things out.

Reply:
Yeah leave it as is in your experiments. Yeah, I regretted the `make_bee` addition since I knew it was likely not that obvious. I should have done it the params way from the beginning, but was convinced this closure method was superior.